service/raft/raft_group0.cc

```diff
@@ -344,7 +344,21 @@ future<> raft_group0::start_server_for_group0(raft::group_id group0_id) {
     assert(group0_id != raft::group_id{});
 
     auto my_id = co_await load_my_id();
-
+    // At this time the group registry is already up and running,
+    // so the address map is getting all the notifications from
+    // the gossiper. By reading the persistent replica of
+    // application state *after* subscribing to new gossip events,
+    // we ensure we haven't missed any IP update in the map.
+    // configuration members.
+    auto ids = co_await db::system_keyspace::load_raft_server_ids();
+    for (auto& id : ids) {
+        auto server_id = raft::server_id{id.second};
+        _raft_gr.address_map().add_or_update_entry(server_id, id.first);
+        _raft_gr.address_map().clear_expiring_flag(server_id);
```

Comment:
I had the same doubts, and I thought that since these addresses are persisted, they must be part of configuration. I will remove this call to set_nonexpiring().